src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -376,10 +366,18 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
         />
       )}
       <AuthenticationModal
-        show={isAuthenticationModalOpen}
+        show={isLoggedIn && isAuthenticationModalOpen}
         onHide={closeAuthenticationModal}
+        headerMessage={"You need an account to join this party"}
         afterUserIsLoggedIn={() => setShouldOpenPaymentModal(true)}
-        showAuth={AuthOptions.register}
+        showAuth={AuthOptions.initial}
```

Comment:
Was this change from `AuthOptions.register` to `AuthOptions.initial` intended? What was the intent/resoning?

Reply:
`initial` shows both register and login buttons and lets the user choose one of them instead of forcing them to register. Just because you are not logged in and you try to join a venue doesn't mean that you don't have an account. This was also in the design when I was writing this thing but it was in trello and it was from a screenshot.

Reply:
Sounds reasonable to me. Thanks for explaining :)